hadoop-ozone/ozonefs-hadoop2/pom.xml

```diff
@@ -53,19 +53,19 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
       <scope>provided</scope>
-      <version>2.7.3</version>
+      <version>3.3.4</version>
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-annotations</artifactId>
       <scope>provided</scope>
-      <version>2.7.3</version>
+      <version>3.3.4</version>
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-auth</artifactId>
       <scope>provided</scope>
-      <version>2.7.3</version>
+      <version>3.3.4</version>
     </dependency>
     <dependency>
       <groupId>ch.qos.reload4j</groupId>
```

Comment:
This submodule is specific to Hadoop 2.  Please do not update it to Hadoop 3.  If necessary, please add an exception or override for the enforcer plugin.

Reply:
Reverted back to 2.7.3 and added exclusions for hadoop-auth and hadoop-annotations